### PR TITLE
Mafia: Fix Crashes

### PR DESCRIPTION
**Pull request by ZestOfLife**

### CODE REVIEW

File: chat-plugins/mafia.js [hunk 1]
```diff
@@ -512,6 +512,10 @@ class MafiaTracker extends Rooms.RoomGame {
 				break;
 			}
 		}
+		if (let p in this.players) {
```
Comment (KrisXV) [inline chat-plugins/mafia.js]: Shouldn’t this be something like `for (const player of Object.keys(this.players))`?

File: chat-plugins/mafia.js [hunk 2]
```diff
@@ -349,7 +349,7 @@ class MafiaTracker extends Rooms.RoomGame {
 		if (this.timer) this.setDeadline('off', true);
 		this.phase = 'night';
 		this.sendRoom(`Night ${this.dayNum}. PM the host your action, or idle.`, {declare: true});
-		if (!early && this.getPlurality()) this.sendRoom(`Plurality is on ${this.players[this.getPlurality()].name}`);
+		if (!early && this.getPlurality() && this.getPlurality() !== 'nolynch') this.sendRoom(`Plurality is on ${this.players[this.getPlurality()].name}`);
```
Comment (HoeenCoder) [inline chat-plugins/mafia.js]: It would be better to simply say that its on `No Lynch` rather than not sending a message at all.

**Comment by HoeenCoder** [comment]
Should of just pulled this to my local repo and tested/fixed it there in the first place...

**Comment by Zarel** [comment]
@HoeenCoder if you're maintaining this, can you rebase?

**Comment by HoeenCoder** [comment]
OK I think this is done right this time. Though I'll let you check @Zarel so I don't screw anything up on the master branch.

**Pull request merged**